thrust/testing/counting_iterator.cu

```diff
@@ -1,3 +1,5 @@
+#define CCCL_IGNORE_DEPRECATED_API
+
```

Comment:
Remark: This may be smarter than suppressing the deprecation warning via pragmas!